app/views/admin/legal_aid_applications/submissions/show.html.erb

```diff
@@ -19,12 +19,7 @@
     <% end %>
     <% if @legal_aid_application&.hmrc_responses.any? %>
       <% @legal_aid_application.hmrc_responses.each do |hmrc_response| %>
-        <details>
-          <summary class="govuk-heading-m govuk-details__summary"
-                   title="HMRC use case <%= hmrc_response.use_case %> <%= hmrc_response.owner_type %>"
-                   aria-label="HMRC use case <%= hmrc_response.use_case %> <%= hmrc_response.owner_type %> result">
-            HMRC response - use case <%= hmrc_response.use_case %> for the <%= hmrc_response.owner_type.downcase %>
-          </summary>
+        <%= govuk_details(summary_text: "HMRC response - use case #{hmrc_response.use_case} for the #{hmrc_response.owner_type.downcase}") do %>
           <section>
```

Comment:
I'm not sure you need the section tags.
I _think_ the do block of govuk_details handles that

Reply:
Thanks, Colin. I just tested it locally and you are right.